javascript/extractor/src/com/semmle/jcorn/ESNextParser.java

```diff
@@ -439,7 +439,10 @@ private DynamicImport parseDynamicImport(Position startLoc) {
     Expression source = parseMaybeAssign(false, null, null);
     Expression attributes = null;
     if (this.eat(TokenType.comma)) {
-      attributes = this.parseMaybeAssign(false, null, null);
+      if (this.type != TokenType.parenR) { // Skip if the comma was a trailing comma
```

Comment:
Optional: Trailing comman in dynamic imports is not part of the tests?

Reply:
Added a trap test